### PR TITLE
feat: Add new commands to manage blob service properties for storage account

### PR DESCRIPTION
**Pull request by jsntcy**
This PR is to support #11423 which added the following two commands:
- `az storage account blob-service-properties show`: Gets the properties of a storage account’s Blob service.
- `az storage account blob-service-properties update --enable-change-feed`: Enable/disable the change feed of a storage account’s Blob service.

TODO: support updating other properties besides 'enable-change-feed'

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
@Juliehzl  please help review the PR.

File: src/azure-cli/azure/cli/command_modules/storage/_params.py [hunk 1]
```diff
@@ -240,6 +240,14 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
         c.argument('vnet_name', help='Name of a virtual network.', validator=validate_subnet)
         c.argument('action', help='The action of virtual network rule.')
 
+    with self.argument_context('storage account blob-service-properties show') as c:
+        c.argument('account_name', acct_name_type, id_part=None)
+
+    with self.argument_context('storage account blob-service-properties update') as c:
+        c.argument('account_name', acct_name_type, id_part=None)
+        c.argument('enable_change_feed', arg_type=get_three_state_flag(),
```
Comment (Juliehzl) [inline src/azure-cli/azure/cli/command_modules/storage/_params.py]: Please give more detailed help message for ChangeFeed feature and use active voice.

File: src/azure-cli/azure/cli/command_modules/storage/commands.py [hunk 2]
```diff
@@ -116,6 +123,15 @@ def get_custom_sdk(custom_module, client_factory, resource_type=ResourceType.DAT
         g.custom_command('list', 'list_network_rules')
         g.custom_command('remove', 'remove_network_rule')
 
+    with self.command_group('storage account blob-service-properties', blob_service_mgmt_sdk,
+                            custom_command_type=storage_account_custom_type,
+                            resource_type=ResourceType.MGMT_STORAGE, min_api='2019-04-01') as g:
```
Comment (Juliehzl) [inline src/azure-cli/azure/cli/command_modules/storage/commands.py]: Blob Service Properties are supported for a long time,  and the correct min api version constraint for blob service properties should be 2018-07-01 I think.
But for changefeed feature it should be 2019-04-01.

**Comment by jiasli** [comment]
Help message for update is missing. 

```
>az storage account blob-service-properties -h

Group
    az storage account blob-service-properties

Commands:
    show   : Gets the properties of a storage account’s Blob service, including properties for
             Storage Analytics and CORS (Cross-Origin Resource Sharing) rules.
    update
```

**Comment by jiasli** [comment]
For `az storage account blob-service-properties show -n {}`, `--group` should be optional, like how `az storage account show` does.

**Comment by jiasli** [comment]
Please run `azdev style storage` to make sure Linter and Style pass.

File: src/azure-cli/azure/cli/command_modules/storage/_params.py (hunk 1 above)
Comment (jsntcy) [reply src/azure-cli/azure/cli/command_modules/storage/_params.py]: Add help in _help.py

---
In reply to: [354645627](https://github.com/Azure/azure-cli/pull/11488#discussion_r354645627) [](ancestors = 354645627)

File: src/azure-cli/azure/cli/command_modules/storage/commands.py (hunk 2 above)
Comment (jsntcy) [reply src/azure-cli/azure/cli/command_modules/storage/commands.py]: updated.

---
In reply to: [354647458](https://github.com/Azure/azure-cli/pull/11488#discussion_r354647458) [](ancestors = 354647458)

**Comment by jsntcy** [comment]
Updated.

---
In reply to: [562439357](https://github.com/Azure/azure-cli/pull/11488#issuecomment-562439357) [](ancestors = 562439357)

**Comment by jsntcy** [comment]
Updated. @jiasli@microsoft.com , please help review again.

---
In reply to: [562439630](https://github.com/Azure/azure-cli/pull/11488#issuecomment-562439630) [](ancestors = 562439630)

File: src/azure-cli/HISTORY.rst [hunk 3]
```diff
@@ -23,13 +23,14 @@ Release History
 
 * GA Release Large File Shares property for storage account create and update command
 * GA Release User Delegation SAS token Support
+* Add new commands to manage blob service properties for storage account.
```
Comment (jiasli) [inline src/azure-cli/HISTORY.rst]: How about we put the command group here: `az storage account blob-service-properties`

Comment (jsntcy) [reply src/azure-cli/HISTORY.rst]: Do we really need put so specific information here?

Comment (jiasli) [reply src/azure-cli/HISTORY.rst]: Not really. We usually refer to the previous history entries.

**Comment by jsntcy** [comment]
@haroldrandom@gmail.com , what's '/azp run' meaning?

---
In reply to: [563201902](https://github.com/Azure/azure-cli/pull/11488#issuecomment-563201902) [](ancestors = 563201902)

**Comment by haroldrandom** [comment]
> @[haroldrandom@gmail.com](mailto:haroldrandom@gmail.com) , what's '/azp run' meaning?
> 
> In reply to: [563201902](https://github.com/Azure/azure-cli/pull/11488#issuecomment-563201902) [](ancestors = 563201902)

Manually rerun Azure DevOps from here.

**Pull request merged**